.github/workflows/frameworks.yml

```diff
@@ -751,7 +751,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install .
-          pip install tensorflow tensorflow_hub transformers jax jaxlib flax torch
+          pip install -U tensorflow tensorflow_hub transformers jax jaxlib flax torch
```

Comment:
Why do we need this?

Reply:
I don't think we need this. Let me remove.